### ISSUE TITLE
what is suitable loss value while train? when should I stop it

### ISSUE THREAD

**Issue by parquets** [matterport/Mask_RCNN#1117]
like the title, I set step 150  and finally I find the loss value is about 0.6, Is the value suitable?

**Comment by stygian2a** [comment]
The value depends entirely on your dataset. I seems good enough, but if your problem is supposed to be a very simple one, then it might not be so good. You should use tensorboard to see the evolution of the training and validation loss. As long as the validation is going down, you keep training. If at some point, it starts to go up while your training loss is still going down (or is staying the same), it means you are overfitting.

**Comment by fastlater** [comment]
Check this: https://github.com/matterport/Mask_RCNN/blob/master/assets/detection_tensorboard.png
As you can see, the loss obtained when training the initial weights was really low so you can estimate that 0.6 is good but maybe not good enough to get art of state results. 
As @stygian2a mentioned, it will highly depends on your dataset, however, lower loss, better results.
